src/storefront/components/Review/RatingsList.tsx

```diff
@@ -0,0 +1,79 @@
+import { IOrder } from "@/types/order";
+import { DataGrid, GridActionsCellItem, GridColDef, GridDeleteIcon } from "@mui/x-data-grid";
+import EditIcon from "@mui/icons-material/Edit";
+import React, { useState } from "react";
+import { useRouter } from "next/router";
+import { Opacity, Visibility } from "@mui/icons-material";
+import { Alert, Box, Card, Divider, Grid, LinearProgress, Rating, Snackbar, Stack, TextField, Typography, alpha } from "@mui/material";
+import StarIcon from '@mui/icons-material/Star';
+import { IReview } from "@/types/review";
+
+interface IReviewsProps {
+    reviews: IReview[];
+}
+
+const labels: { [index: string]: string } = {
+    10: 'Useless',
+    20: 'Useless+',
+    30: 'Poor',
+    40: 'Poor+',
+    50: 'Ok',
+    60: 'Ok+',
+    70: 'Good',
+    80: 'Good+',
+    90: 'Excellent',
+    100: 'Excellent+',
+};
```

Comment:
We should maybe extract the labels to separate method, in order to keep our code DRY, what do you think?

Reply:
Changed.